doc/glossary.rst

```diff
@@ -42,6 +42,11 @@
                 Ceph is a distributed network storage and file system with
                 distributed metadata management and POSIX semantics.
 
+        `ceph-ansible <https://docs.ceph.com/projects/ceph-ansible/en/latest/index.html>`_
+                A github repository, supported from the Jewel release to the
```

Comment:
GitHub ?

Reply:
Yeah, let's go camelcase.

Reply:
It's how they style it, so that seems appropriate.